conformance/lang/expressions/relational-expr/less_than_expression.balt

```diff
@@ -0,0 +1,1225 @@
+Labels: relational-expr, string
+
+function init() {
+   string a = "ABCD";
+   string b = "ABCDE";
+   string c = "DCBA";
+   io:println(a < b); // @output true
+   io:println(b < a); // @output false
+   io:println(a < c); // @output true
+   io:println(c < a); // @output false
+   io:println(c < c); // @output false
+}
+
+Test-Case: output
+Description: Test less than expression when one operand is optional string
+Fail-Issue: ballerina-platform/ballerina-lang#35195
+Labels: relational-expr, optional-type, string
+
+function init() {
+   string? a = "ABCD";
+   string b = "ABCDE";
+   string? c = "DCBA";
+   io:println(a < b); // @output true
+   io:println(b < a); // @output false
+}
+
+Test-Case: output
+Description: Test less than expression when both operands are string
+Labels: relational-expr, optional-type, string
+
+function init() {
+   string? a = "ABCD";
+   string? b = "ABCDE";
+   string? c = "DCBA";
+   io:println(a < b); // @output true
+   io:println(b < a); // @output false
+   io:println(a < c); // @output true
+   io:println(c < a); // @output false
+   io:println(c < c); // @output false
+}
+
+Test-Case: output
+Description: Test less than expression when both operands are positive decimals
+Labels: decimal, decimalNumber, relational-expr
+
+function init() {
+   decimal a = 32.45;
+   decimal b = 32;
+   decimal c = 3.12345678888888888888888888888889999999;
+   decimal d = 3.1234567888888888888888888888888999999900000000000000000000000000000000000000000000001;
+   io:println(a < b); // @output false
+   io:println(b < a); // @output true
+   io:println(c < d); // @output false
+   io:println(d < c); // @output false
+}
+
+Test-Case: output
+Description: Test less than expression when both operands are negative decimals
+Labels: decimal, decimalNumber, relational-expr, unary-minus
+
+function init() {
+   decimal a = -32.45;
+   decimal b = -32;
+   decimal c = -3.12345678888888888888888888888889999999;
+   decimal d = -3.1234567888888888888888888888888999999900000000000000000000000000000000000000000000001;
+   io:println(a < b); // @output true
+   io:println(b < a); // @output false
+   io:println(c < d); // @output false
+   io:println(d < c); // @output false
+}
+
+Test-Case: output
+Description: Test less than expression when one operand is positive decimal and other one is negative
+Labels: decimal, decimalNumber, relational-expr, unary-minus
+
+function init() {
+   decimal a = -32.45;
+   decimal b = 32;
+   decimal c = -3.12345678888888888888888888888889999999;
+   decimal d = 3.1234567888888888888888888888888999999900000000000000000000000000000000000000000000001;
+   io:println(a < b); // @output true
+   io:println(b < a); // @output false
+   io:println(c < d); // @output true
+   io:println(d < c); // @output false
+}
+
+Test-Case: output
+Description: Test less than expression when both operands are positive floats
+Labels: decimalNumber, float, relational-expr
```

Comment:
Should be `DecimalNumber`. Fix in other places too.
Missing `DecimalFloatingPointNumber` and `multiplicative-expr` labels here